docs/voltohandson/starttheming.md

```diff
@@ -38,25 +38,25 @@ Everytime you add a new file to your project you will have to restart your devel
 Edit the new file and add this:
 
 ```less
-@fontName: "Asisstant";
+@fontName: "Assistant";
 ```
 
 You can set it to any Google font available, and the online version of the font will be used.
 You can also set other variables concerning the font used, such as the sizes available.
 In case you want to use more than one font or a font that is self-hosted,
-you should define it as usual in CSS and set the variable `importGoogleFonts` appropriately. As `Assisstant` is Google Font we will set:
+you should define it as usual in CSS and set the variable `importGoogleFonts` appropriately. As `Assistant` is Google Font we will set:
 
 ```less
 @importGoogleFonts: true;
 ```
 
-Two more important variables that are changed in plone.org are:
+Two more important Variables that are changed in plone.org are:
 
 ```less
 @largeMonitorBreakpoint: 1330px;
 @emSize: 16px;
 ```
-This changes the width of the default container. Add them as well.
+These Two Variables change the Width of the Default Container. Add them as well.
```

Comment:
Why So Many Title Case Words? This Is Not A Newspaper Headline. 😉 

Also we prefer one sentence per line for easier diffs, but that's a minor thing.

```suggestion
These two variables change the width of the default container.
Add them as well.
```